packages/modules/viewer-react/package.json

```diff
@@ -33,31 +33,34 @@
     "@itwin/itwinui-illustrations-react": "^2.0.1",
     "@itwin/itwinui-react": "^2.6.0",
     "@itwin/reality-data-client": "^1.0.0",
+    "@itwin/unified-selection": "^0.4.4",
     "lodash.isequal": "^4.5.0"
   },
   "devDependencies": {
-    "@itwin/appui-abstract": "^4.0.0",
-    "@itwin/appui-layout-react": "^4.0.0",
-    "@itwin/appui-react": "^4.0.0",
-    "@itwin/build-tools": "^4.0.0",
-    "@itwin/components-react": "^4.0.0",
-    "@itwin/core-bentley": "^4.0.0",
-    "@itwin/core-common": "^4.0.0",
-    "@itwin/core-frontend": "^4.0.0",
-    "@itwin/core-geometry": "^4.0.0",
-    "@itwin/core-i18n": "^4.0.0",
-    "@itwin/core-markup": "^4.0.0",
-    "@itwin/core-orbitgt": "^4.0.0",
-    "@itwin/core-quantity": "^4.0.0",
-    "@itwin/core-react": "^4.0.0",
-    "@itwin/core-telemetry": "^4.0.0",
-    "@itwin/ecschema-metadata": "^4.0.0",
+    "@itwin/appui-abstract": "^4.7.3",
+    "@itwin/appui-react": "4.15.0",
+    "@itwin/build-tools": "^4.7.3",
+    "@itwin/components-react": "4.15.0",
+    "@itwin/core-bentley": "^4.7.3",
+    "@itwin/core-common": "^4.7.3",
+    "@itwin/core-frontend": "^4.7.3",
+    "@itwin/core-geometry": "^4.7.3",
+    "@itwin/core-i18n": "^4.7.3",
+    "@itwin/core-markup": "^4.7.3",
+    "@itwin/core-orbitgt": "^4.7.3",
+    "@itwin/core-quantity": "^4.7.3",
+    "@itwin/core-react": "4.15.0",
+    "@itwin/core-telemetry": "^4.7.3",
+    "@itwin/ecschema-metadata": "^4.7.3",
+    "@itwin/ecschema-rpcinterface-common": "^4.7.3",
```

Comment:
are we really able to only add these new deps as just dev deps? they dont need to be peers?

Reply:
Nice catch. It looks like `"@itwin/ecschema-rpcinterface-common"` should be peer dep. Although I think it would be nice to avoid it. Maybe instead of creating `SchemaContext` in viewer we could take a prop like `getSchemaContext?: (imodel: IModelConnectio) => SchemaContext`. This would allow to reuse same SchemaContext between Viewer and other components. 

Also these deps should be moved under `dependencies`:
```
"@itwin/presentation-core-interop": "^0.2.3",
"@itwin/presentation-shared": "^0.3.1",
```

Reply:
Added a `getSchemaContext` prop instead of creating `SchemaContext` and moved the dependencies.